docs/FRSKY_R9.md

```diff
@@ -70,31 +88,94 @@ The wiring is similar to the ExpressLRS docs found in the link below. Different
 
+If you want to use ELRS bootloader and install via the Frsky bootloader and OpenTX or EdgeTX, the R9MM is the smallest and lightest receiver available.  Note: The R9MM has a different [wiring diagram](https://www.expresslrs.org/quick-start/receivers/r9/#wiring-up-your-receiver) than the R9MX.
 
 ## Flashing ##
 
-ExpressLRS has figured out a convenient and easy way for flashing, which is unfortunately not available for mLRS (someone needs to figure it out). For flashing mLRS only the grass-route DIY procedure using a ST-Link programmer is currently possible.
+### Acknowledgments ###
 
-Note: Fashing mLRS with the ST-Link is a non-reversible operation, i.e., it is not possible to revert back to the original Frsky firmware. It is possible to switch to ExpressLRS however.
+We wish to express our thanks to the folks of the ExpressLRS project, who have worked out the easy way to flash the R9 hardware, which is now also available for mLRS. With gratitude, here we are utilizing the ExpressLRS bootloader images and scripts and, in several cases, also their excellent documentation.
 
-In principle, the procedure goes exactly as already described in the ExpressLRS docs:
-- R9M module: https://www.expresslrs.org/1.0/quick-start/tx-r9m/#flashing-using-stlink
-- R9MX receiver: https://www.expresslrs.org/1.0/quick-start/rx-stlink/
+### Flash the R9M with ELRS bootloader ###
+
+You can flash the ELRS bootloader and mLRS using your radio with OpenTX 2.3.12 or newer or EdgeTX 2.4.0 or newer.  This method is especially recommended for new users.
+
+Of course, flashing via ST-Link is also still an option.
+
+The ExpressLRS documentation provides ELRS specific instructions [here](https://www.expresslrs.org/quick-start/transmitters/frsky-r9modules/#via-stock_bl) which you might want to reference if the instructions here are not clear.  Of course, you should ignore the references to the ELRS configurator and build mLRS as instructed elsewhere.
+
+#### Flash the ELRS bootloader ####
+
+These steps only need to be performed once.  If you experience a "No Sync" error, check that you have selected CRSF external mode and not mBridge.
+
+If you have never previously flashed this module via ST-Link, you can install the ELRS bootloader using and alongside the stock bootloader which also preserves the ability to return to the stock Frsky firmware.
+
+1. Download the [r9m\_elrs\_bl.frk](https://github.com/ExpressLRS/ExpressLRS/blob/master/src/bootloader/r9m_elrs_bl.frk?raw=true) file from the ExpressLRS git repository and copy it to the FIRMWARE folder on your radio's SD card.
+
+2. Install the R9M module in your radio.  Enter the System Menu and navigate using the page buttons to the SD card page.
```

Comment:
- I guess I would put the 2nd sentence either as separate bullet, but probably would actually add it to the next bullet.
- In this 2nd sentence I woudl somehow reference to that it's the System Menu in the radio.